src/DynamoCoreWpf/ViewModels/Search/NodeSearchElementViewModel.cs

```diff
@@ -305,8 +306,9 @@ private void AutoLayoutNodes(object sender, EventArgs e)
         {
             var nodeView = (NodeView) sender;
             var dynamoViewModel = nodeView.ViewModel.DynamoViewModel;
+            var originalNodeId = nodeView.ViewModel.NodeModel.OutputNodes.Values.SelectMany(s => s.Select(t => t.Item2)).Distinct().FirstOrDefault().GUID;
```

Comment:
how about
```
var originalNodeId = nodeView.NodeViewModel.Id;
```

Reply:
Here we need Id for the original node that had the node autocomplete triggered on it, the suggested `nodeView.ViewModel.Id` will give us Id of the new selected node instead.

Reply:
Ok, but this seems like a really roundabout way of getting the original node.

Reply:
it is, indeed 😅